Current `main` branch breaks FlowForge build

As of #151 being merged, the main flowforge app no longer builds. I'm pretty sure I've not missed a step here.

```
# In this repo:
git pull
npm install
npm run build
```

Then, in flowforge repo (with ui-components sym-linked into node_modules):
```
npm run serve
```


```
ERROR in ../forge-ui-components/dist/forge-ui-components.js 7:16-51
Module not found: Error: Can't resolve '@/components/Spinner.vue' in '/Users/nol/code/ff/flowforge-dev-env/packages/forge-ui-components/dist'
resolve '@/components/Spinner.vue' in '/Users/nol/code/ff/flowforge-dev-env/packages/forge-ui-components/dist'
  Parsed request is a module
  using description file: /Users/nol/code/ff/flowforge-dev-env/packages/forge-ui-components/package.json (relative path: ./dist)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      /Users/nol/code/ff/flowforge-dev-env/packages/forge-ui-components/dist/node_modules doesn't exist or is not a directory
      looking for modules in /Users/nol/code/ff/flowforge-dev-env/packages/forge-ui-components/node_modules
        /Users/nol/code/ff/flowforge-dev-env/packages/forge-ui-components/node_modules/@/components doesn't exist
      /Users/nol/code/ff/flowforge-dev-env/packages/node_modules doesn't exist or is not a directory
      looking for modules in /Users/nol/code/ff/flowforge-dev-env/node_modules
        /Users/nol/code/ff/flowforge-dev-env/node_modules/@/components doesn't exist
      /Users/nol/code/ff/node_modules doesn't exist or is not a directory
      /Users/nol/code/node_modules doesn't exist or is not a directory
      /Users/nol/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
 @ ./frontend/src/main.js 2:0-63 12:31-48
```


_Originally posted by @knolleary in https://github.com/flowforge/forge-ui-components/issues/151#issuecomment-1638455368_

--- a/src/components.js
+++ b/src/components.js
@@ -33,6 +33,7 @@ import FFTab from './components/tabs/Tab.vue'
 import FFCheck from './components/Check.vue'
 import FFListItem from './components/ListItem.vue'
 import FFMarkdownViewer from './components/Markdown.vue'
+import FFSpinner from './components/Spinner.vue'
 
 export default {
     FFButton,
@@ -42,6 +43,7 @@ export default {
     FFListItem,
     FFCheck,
     FFMarkdownViewer,
+    FFSpinner,
     // Data Table
     FFDataTable,
     FFDataTableRow,

--- a/src/components/form/ToggleSwitch.vue
+++ b/src/components/form/ToggleSwitch.vue
@@ -4,20 +4,15 @@
         <div class="ff-toggle-switch-slider" @click="toggle">
             <div class="ff-toggle-switch-button">
                 <slot v-if="!loading"></slot>
-                <FFSpinner v-else />
+                <ff-spinner v-else />
             </div>
         </div>
     </label>
 </template>
 
 <script>
-import FFSpinner from '@/components/Spinner.vue'
-
 export default {
     name: 'ff-toggle-switch',
-    components: {
-        FFSpinner
-    },
     props: {
         disabled: {
             default: false,

